docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -77,17 +90,61 @@ const finalTheme = createTheme({
 
 {{"demo": "GlobalThemeOverrideCallback.js"}}
 
-### Using `sx` (experimental) syntax
+### Using the `sx` experimental syntax
+
+If you are not familiar with the `sx` prop, make sure you first check out [the concept around it](/system/the-sx-prop/) and [the difference with the `styled` utility](/system/styled/#difference-with-the-sx-prop).
+
+To have a similar shorthand CSS notation to the `sx` prop but within the theme, you can use `sx` inside the `stylesOverrides`.
+
+{{"demo": "GlobalThemeOverrideSx.js", "defaultCodeOpen": false}}
+
+```tsx
+const finalTheme = createTheme({
+  components: {
+    MuiChip: {
+      styleOverrides: {
+        root: sx({
+          px: 1,
+          py: 0.25,
+          borderRadius: 1,
+        }),
+        label: {
+          padding: 'initial',
+        },
+        icon: sx({
+          mr: 0.5,
+          ml: '-2px',
+        }),
+      },
+    },
+  },
+});
+```
+
+### Specificity
 
-If you are not familiar `sx`, first check out [the concept](/system/the-sx-prop/) and [the difference with the `styled`](/system/styled/#difference-with-the-sx-prop).
+If you use the theming approach to customize the components, you'll still be able to override them using the `sx` prop as it has a higher CSS specificity.
 
-`sx` is also compatible with theme style overrides if you prefer the shorthand notation.
+Note that some components expose a `color` prop, which has a lower specificty than the theme's styles overrides. For example, say you have customized `Link`'s color.
```

Comment:
@danilo-leal I was wrong. The `Link` is actually a bug. This should have the highest priority `<Link color="highest-priority-color">...</Link>`.

Reply:
Makes sense! I was just thinking it was weird for the component color prop to have lower specificity... I'll correct this sentence!

Reply:
@siriwatknp are you going to bump the specificity for each prop combination? Should we create an RFC for this and make it work consistent with Material UI? In this case, when would you ever need the props callback?

Reply:
@mnajdova I am not sure I follow you. From my understanding, this is a bug on the Link component.